packages/core/theme/src/components/date-picker.ts

```diff
@@ -13,8 +13,8 @@ const datePicker = tv({
     selectorButton: "-mr-2 text-inherit",
     selectorIcon: "text-lg text-inherit pointer-events-none flex-shrink-0",
     popoverContent: "p-0 w-full",
-    calendar: "w-[var(--calendar-width)] shadow-none",
-    calendarContent: "w-[var(--calendar-width)]",
+    calendar: "w-[256px] shadow-none",
+    calendarContent: "w-[256px]",
```

Comment:
Please update the section where `--calendar-width` is set.

https://github.com/nextui-org/nextui/blob/ec7c4173875f02981182a3b513936c15a698ad06/packages/components/calendar/src/use-calendar-base.ts#L274

It should look like this:

```tsx
"--calendar-width": typeof calendarWidth === "number" ? `${calendarWidth}px` : calendarWidth,
```

Additionally, it would be gread if you could add a Storybook example where the `calendarWidth` prop is specified as a number.